### ISSUE TITLE
Serde-brief spec license

### ISSUE THREAD

**Issue by jamesmunns** [FlixCoder/serde-brief#7]
Hey there!

I just wanted to note that the postcard wire spec document is [licensed under CC-BY-SA 4.0](https://postcard.jamesmunns.com/#license). This license requires that any remixed works:

* Use the same CC-BY-SA license
* Attribute the original work

It *looks* like the Serde-Brief spec is based on the Postcard spec.

If it is, I'd appreciate if you could do the two items above! If it's not (I have no idea where the line is between "inspired by" and "actual remix" is), then feel free to close. I trust your judgement to decide which of those it is, but it's something I noticed.

**Comment by FlixCoder** [comment]
Hi!

I did notice the license, though I took care not to copy anything and only influenced myself on the document structure. I wrote everything out of my head with my own words, also to bring across the intentions in design I had with the library. As I said before, I would certainly say it is influenced, but I would not call it a remix, though I don't know the exact definitions.

I am happy to add more attribution, since the structure helped. If you feel like anything is too close or similar, I am happy to discuss the specific sections and change them, though I probably want to keep it separate from your license (though I don't know your exact terms, maybe I would agree with the license, but your docs link to 404 instead of the license btw).

That said: Postcard was not the biggest influence, there are other similar libraries and formats and serde also just has a fixed framework. But I do value the great docs and example and took some small ideas.

I am sorry if you feel like I copied something without valueing the authorship! Feel free to detail where and how you imagine the attribution knowing what I just tried to describe
